src/ape/api/address.py

```diff
@@ -148,9 +132,6 @@ def codesize(self) -> int:
     def is_contract(self) -> bool:
         """
         ``True`` when there is code associated with the address.
-
-        Returns:
```

Comment:
`Returns` looks back for `property` docs, makes it look like you have to call it